storage_service/locations/models/event.py

```diff
@@ -100,6 +106,20 @@ class Callback(models.Model):
         verbose_name=_("Method"),
         help_text=_("HTTP request method to use in connecting to the URL."),
     )
+    body = models.TextField(
```

Comment:
I can see a case both ways, but I'm not sure yet if we actually need to be configurable. It may be more useful to have a standard webhook payload that we document.

Reply:
The idea of making it configurable was to facilitate SS upgrades and consumer integrations. For example, in SCOPE I'm checking if a download URL is present in the payload and falling back to build a SS API v2 URL with the DIP uuid and the Origin URL.

Reply:
I think the main problem here is if you are a third party writing a consumer...how do you know what to expect in the payload? But maybe we'll end up needing the configurability. NHA might even.